packages/shared/src/lib/auxiliary/wallet-connect/handlers/eth_transaction.handler.ts

```diff
@@ -68,6 +63,6 @@ export async function handleEthTransaction(
             },
         })
     } catch (err) {
-        responseCallback({ error: getSdkError(err) })
+        responseCallback(getBloomError(err))
```

Comment:
in `await switchToRequiredAccount(from, chain)` we potentially throw `UNSUPPORTED_ACCOUNTS` or `USER_REJECTED` which are walletconnect specific errors. For these we need the `getSdkError`